pallets/msa/src/tests.rs

```diff
@@ -341,14 +341,14 @@ fn it_deletes_msa_last_key_self_removal() {
 fn test_retire_msa_success() {
 	new_test_ext().execute_with(|| {
 		let (test_account_key_pair, _) = sr25519::Pair::generate();
-		let msa_id = 2;
 
 		// Create an account
 		let test_account = AccountId32::new(test_account_key_pair.public().into());
 		let origin = Origin::signed(test_account.clone());
 
-		// Add an account to the MSA so it has exactly one account
-		assert_ok!(Msa::add_key(msa_id, &test_account, EMPTY_FUNCTION));
+		// Create an MSA so this account has one key associated with it
+		assert_ok!(Msa::create(origin.clone()));
```

Comment:
It's a subtle change but since I had forgotten that you could just stuff a fake MSA into the registry directly, it profoundly confused me.  So I changed it to call the extrinsic instead.  🤷‍♀️